<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Сторона 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Сторона 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_size_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_size_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32766</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_angle">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_in_degrees">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>в градусах</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_in_radians">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>в радианах</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_answer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OK">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>74</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>74</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
